plotly/offline/offline.py

```diff
@@ -185,3 +165,12 @@ def plot():
     """ Configured to work with localhost Plotly graph viewer
     """
     raise NotImplementedError
+
+
+try:
+    _init_notebook_mode()
```

Comment:
also, that's what test suites are for! let's make sure that those things aren't cached in circle and we'll be fine!